### PR TITLE
Update STYLE_GUIDE.md

### PR DESCRIPTION
**Pull request by goldmermaid**
From publisher requirements (more details in PR #868)

### CODE REVIEW

File: STYLE_GUIDE.md [hunk 1]
```diff
@@ -17,6 +17,9 @@
     * 时刻t（不是t时刻）
 	* 形状使用英文标点
         * (10, 20) 不是 （10，20）
+* 空格：
+	* 文本中中文和英文、数字、特殊字体等之间不要空格，除了`:numref:`、`:cite:`等前留有一个英文空格（否则网页不渲染）
```
Comment (zhuyuanxiang) [inline STYLE_GUIDE.md:21]: 中文与数学公式之间是否也不加空格呢？

Comment (goldmermaid) [reply STYLE_GUIDE.md]: `:numref:`、`:cite:`前加空格就可渲染了

Comment (goldmermaid) [reply STYLE_GUIDE.md]: great catch, added! thanks!

**Comment by mli** [comment]
Job d2l-zh/PR-889/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-889/

**Comment by mli** [comment]
Job d2l-zh/PR-889/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-889/

**Comment by mli** [comment]
Job d2l-zh/PR-889/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-889/

File: STYLE_GUIDE.md (hunk 1 above)
Comment (zhuyuanxiang) [reply STYLE_GUIDE.md]: :-)

**Pull request merged**